app/app_main.c

```diff
@@ -2142,7 +2182,7 @@ static int enable_kas_ffc(ACVP_CTX *ctx) {
     CHECK_ENABLE_CAP_RV(rv);
     rv = acvp_cap_kas_ffc_set_parm(ctx, ACVP_KAS_FFC_SSC, ACVP_KAS_FFC_MODE_NONE, ACVP_KAS_FFC_GEN_METH, ACVP_KAS_FFC_FB);
     CHECK_ENABLE_CAP_RV(rv);
-#if OPENSSL_VERSION_NUMBER < 0x30000000L
+#if OPENSSL_VERSION_NUMBER >= 0x30100000L
```

Comment:
originally you wanted an old version, and now you want the newest version ?

Reply:
Correct. Previously, we wanted to only use with FOM and 1.X.X then skip 3.0 since it wasnt on the 3.0 cert, but now its on the 3.1.2 cert :) some fragments of 1.X.X support remain but I am slowly stripping them out.